NEWS.md

```diff
@@ -4,7 +4,7 @@ there might not be any notable changes in new versions of this project.
 # NEWS
 
 ## 6.4.4 (October 25, 2024)
-
+* Supporting Rails 7.2.
```

Comment:
I'm removing this line to merge this PR right away. This is going to be released later today.
```suggestion
```